packages/manager/apps/dedicated/client/app/ip/ip.controller.js

```diff
@@ -26,7 +27,8 @@ export default /* @ngInject */ function IpMainCtrl(
   openBannerRepricePage,
 ) {
   $scope.currentUser = currentUser;
-  $scope.goToOrganisation = () => goToOrganisation();
+  $scope.goToOrganisation = (vrackId) => goToOrganisation(vrackId);
+  $scope.goToVrack = () => goToVrack();
```

Comment:
Why creating a new function, you can keep reference to the existing one

```suggestion
  $scope.goToOrganisation = goToOrganisation;
  $scope.goToVrack = goToVrack;
```

Reply:
I do it like goToOrganisation but you're right I agree with you ;)